queries.sql

```diff
@@ -137,4 +137,63 @@ FROM owners
     JOIN animals ON animals.owners_id = owners.id
 GROUP BY owners.id
 ORDER BY num_of_animals DESC
-LIMIT 1;
\ No newline at end of file
+LIMIT 1;
+
```

Comment:
**[OPTIONAL REQUIREMENTS]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍